src/settings.py

```diff
@@ -13,7 +13,9 @@
     cast=lambda filename: tomllib.loads(Path(filename).read_text()),
     default={},
 )
-LLM_CONFIG = PROJECT_CONFIG.get("llm") or {"temperature": 0.2}
+
+DEFAULT_LLM_CONFIG = {"temperature": 0.2, "model_name": "gpt-3.5-turbo"}
+LLM_CONFIG = {**DEFAULT_LLM_CONFIG, **PROJECT_CONFIG.get("llm", {})}
```

Comment:
As I commented before, this makes it harder to read the code. For those unexperienced with Python, it's hard to understand that this code is actually an update of the dictionary using dict-unpacking.

I'd rather have it more explicit than a single-liner

Reply:
a more readable way to merge these dicts could be this, however a bit more verbose:
```
DEFAULT_LLM_CONFIG = {"temperature": 0.2, "model_name": "gpt-3.5-turbo"}
LLM_CONFIG = DEFAULT_LLM_CONFIG.copy()
LLM_CONFIG.update(PROJECT_CONFIG.get("llm", {}))
```
What do you think @walison17 @vmesel ?

Reply:
@lgabs LGTM